locales/it_IT.json

```diff
@@ -563,57 +563,57 @@
     }
   },
   "account-setup": {
-    "tab-title": "Imposta account",
+    "tab-title": "Set up account",
```

Comment:
@BlueWillow91 This PR is reverting strings back to English. Could you rebase and try again?

Reply:
Since it is just one line, it may be faster to scrap this PR and just do another one. 

I only merged in your most recent changes this morning.

Reply:
@lexipenia
Thank you, I'll scrap this one.
I'm sorry if I'm making it messy. There's a lot I could learn or figure out yet.

Could you point to me the quickest way to do what you described, please?